test/Microsoft.AspNet.Routing.Tests/RouteValueDictionaryTests.cs

```diff
@@ -168,6 +168,48 @@ public void CreateFromObject_MixedCaseThrows()
                 expected);
         }
 
+        [Fact]
+        public void CreateFromReadOnlyDictionary_CopiesValues()
+        {
+            // Arrange
+            var dictionary = new Dictionary<string, object>();
+            dictionary.Add("Name", "James");
+            dictionary.Add("Age", 30);
+            var readonlyDictionary = (IReadOnlyDictionary<string, object>)dictionary;
+
+            // Act
+            var dict = new RouteValueDictionary(readonlyDictionary);
+
+            // Assert
+            Assert.Equal(2, dict.Count);
+            Assert.Equal("James", dict["Name"]);
+            Assert.Equal(30, dict["Age"]);
+        }
+
+        [Fact]
+        public void CreateFromReadOnlyDictionary_ModifyRouteValueDictionary()
```

Comment:
what are the expectations of this test?  is it more like `_Add_ModifiesRouteValueDictionary`?